### PR TITLE
channel transfers multi tests

### PR DESCRIPTION
**Pull request by zemtsov**

### CODE REVIEW

File: test/integration/channel-transfer-multi/channel_transfer_multi_suit_test.go [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+	Expect(err).NotTo(HaveOccurred())
+})
+
+var _ = SynchronizedAfterSuite(func() {
+}, func() {
+	buildServer.Shutdown()
+})
+
+func StartPort() int {
+	return integration.GossipBasePort.StartPortForNode()
```
Comment (pfi79) [inline test/integration/channel-transfer-multi/channel_transfer_multi_suit_test.go:46]: Change the port to KafkaBasePort

File: test/integration/channel-transfer-multi/channel_transfer_multi_test.go [hunk 2]
```diff
@@ -0,0 +1,495 @@
...
+				"industrialBalanceOf", user.AddressBase58Check)
+		}
+	})
+
+	It("multi transfer by admin test", func() {
+		By("creating grpc connection")
+		clientCtx = metadata.NewOutgoingContext(context.Background(), metadata.Pairs("authorization", networkFound.ChannelTransfer.AccessToken))
+
+		transportCredentials := insecure.NewCredentials()
+		grpcAddress := networkFound.ChannelTransfer.HostAddress + ":" + strconv.FormatUint(uint64(networkFound.ChannelTransfer.Ports[cmn.GrpcPort]), 10)
```
Comment (pfi79) [inline test/integration/channel-transfer-multi/channel_transfer_multi_test.go:324]: It seems that there are functions in standard libraries that make address from host and port

File: test/integration/channel-transfer-multi/channel_transfer_multi_test.go [hunk 3]
```diff
@@ -0,0 +1,495 @@
...
+				"allowedBalanceOf", user.AddressBase58Check, transferItems[i].GetToken())
+		}
+	})
+
+	It("multi transfer by customer test", func() {
+		By("creating grpc connection")
+		clientCtx = metadata.NewOutgoingContext(context.Background(), metadata.Pairs("authorization", networkFound.ChannelTransfer.AccessToken))
+
+		transportCredentials := insecure.NewCredentials()
+		grpcAddress := networkFound.ChannelTransfer.HostAddress + ":" + strconv.FormatUint(uint64(networkFound.ChannelTransfer.Ports[cmn.GrpcPort]), 10)
```
Comment (pfi79) [inline test/integration/channel-transfer-multi/channel_transfer_multi_test.go:413]: It seems that there are functions in standard libraries that make address from host and port

File: test/integration/channel-transfer-multi/channel_transfer_multi_suit_test.go (hunk 1 above)
Comment (zemtsov) [reply test/integration/channel-transfer-multi/channel_transfer_multi_suit_test.go]: Done

File: test/integration/channel-transfer-multi/channel_transfer_multi_test.go (hunk 3 above)
Comment (zemtsov) [reply test/integration/channel-transfer-multi/channel_transfer_multi_test.go]: Function `net.JoinHostPort` used

**Review by pfi79 — APPROVED** [review]

**Review by pfi79 — APPROVED** [review]

**Pull request merged**